src/map.react.js

```diff
@@ -505,21 +504,33 @@ export default class MapGL extends Component {
     this._onMouseDown({pos});
   }
 
+  @autobind _onTouchDrag({pos}) {
```

Comment:
Do we need to use de-structuring here vs. just passing the entire argument as `opt` through?